packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -139,7 +210,9 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
   function getOrComputeTobinTax() external nonReentrant returns (uint256, uint256) {
     // solhint-disable-next-line not-rely-on-time
     if (now.sub(tobinTaxCache.timestamp) > tobinTaxStalenessThreshold) {
-      tobinTaxCache.numerator = uint128(computeTobinTax());
+      FixidityLib.Fraction memory denominator = FixidityLib.newFixed(TOBIN_TAX_DENOMINATOR);
```

Comment:
I think this can be simplified to:

`tobinTaxCache.numerator = computeTobinTax();`